tctl.yaml

```diff
@@ -37,11 +37,20 @@ pipeline:
       go mod tidy
 
       make build
-      install -Dm755 tctl ${{targets.destdir}}/usr/bin/tctl
-      install -Dm755 tctl-authorization-plugin ${{targets.destdir}}/usr/bin/tctl-authorization-plugin
+      install -Dm755 tctl ${{targets.destdir}}/usr/local/bin/tctl
+      install -Dm755 tctl-authorization-plugin ${{targets.destdir}}/usr/local/bin/tctl-authorization-plugin
```

Comment:
I think we no longer need this since we are already adding a sub package and installing it there

Reply:
smh, true